src/Scriban/ScribanAsync.generated.cs

```diff
@@ -1289,9 +1288,17 @@ public static async ValueTask<object> CallAsync(TemplateContext context, ScriptN
 
             var scriptFunction = functionObject as ScriptFunction;
             var function = functionObject as IScriptCustomFunction;
+            var isPipeCall = processPipeArguments && context.CurrentPipeArguments != null && context.CurrentPipeArguments.Count > 0;
             if (function == null)
             {
-                throw new ScriptRuntimeException(callerContext.Span, $"Invalid target function `{functionObject}` ({context.GetTypeName(functionObject)})");
+                if ((isPipeCall) && (callerContext is ScriptFunctionCall funcCall))
```

Comment:
These changes are coming from a different PR that were merged. So it doesn't look like you are on master.

Reply:
Hmm... Actually I was on master. This code was generated (changed) when I run Scriban.AsyncCodeGen (together with line 859 in ScribanAsync.generated.cs). I assumed this code generator was not run lately, so it changed not only the line I wanted, but also other lines that were not up-to-date. Am I wrong?

Reply:
Oh, ok, maybe that's the issue, the original PR forgot to make the `AsyncCodeGen` and that was missed
FYI @binarycow

Reply:
Yeah... sorry @xoofx... I forgot about the code gen...

I'm curious tho... the build task in the original PR (#354) passed.  @szyb says that `master` was failing to build.  Shouldn't that not be possible, if the build task passes before its merged?

Also, what are your thoughts about adding the codegen/docgen process to the CI/CD pipeline?

Reply:
@binarycow Failing build after merge was because other PR #353 (which was later merged to the master than your base commit). It added method `GetValueToSet` in `StringAssignExpression` which uses `ScriptBinaryOperator.Substract` (with the typo) and PR #354 renamed this enum value. After merge it failed to build.

I think it might be a good idea to generate the code/docs in CI/CD.

Reply:
> I think it might be a good idea to generate the code/docs in CI/CD.

Yes, as a verification test. I don't want the CI to generate extra commits on top of users ones, but we do want to catch that if someone made a change and forgot to update/run the file, we can detect this with a unit test.

I need to add a message for PR contribution so that also folks making a PR don't forget these steps when they contribute. It's currently not written anywhere, so it's really easy to miss them - and for me to forget about them during PR reviews.